Whithout the `prePatch` block, I get the following error on Python 3:
```
======================================================================
ERROR: test (unittest.loader._FailedTest)
----------------------------------------------------------------------
ImportError: Failed to import test module: test
Traceback (most recent call last):
  File "/nix/store/4wqa1whb42khic67rv9jgip30nxmaii8-python3-3.6.4/lib/python3.6/unittest/loader.py", line 153, in loadTestsFromName
    module = __import__(module_name)
  File "/tmp/nix-build-python3.6-fritzconnection-0.6.5.drv-0/fritzconnection-0.6.5/fritzconnection/test.py", line 16, in <module>
    from fritzconnection import (
ImportError: cannot import name 'FritzXmlParser'
```
But as it is, building and testing works fine on both, Python 2 and 3.

--- a/pkgs/development/python-modules/fritzconnection/default.nix
+++ b/pkgs/development/python-modules/fritzconnection/default.nix
@@ -1,0 +1,25 @@
+{ stdenv, buildPythonPackage, fetchPypi, lxml, requests, tkinter }:
+
+buildPythonPackage rec {
+  pname = "fritzconnection";
+  version = "0.6.5";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "14g3sxprq65lxbgkf3rjgb1bjqnj2jc5p1swlq9sk9gwnl6ca3ds";
+  };
+
+  prePatch = ''
+    substituteInPlace fritzconnection/test.py \
+      --replace "from fritzconnection import" "from .fritzconnection import"
+  '';
+
+  propagatedBuildInputs = [ lxml requests tkinter ];
+
+  meta = with stdenv.lib; {
+    description = "Python-Tool to communicate with the AVM FritzBox using the TR-064 protocol";
+    homepage = https://bitbucket.org/kbr/fritzconnection;
+    license = licenses.mit;
+    maintainers = with maintainers; [ dotlambda ];
+  };
+}

--- a/pkgs/top-level/python-packages.nix
+++ b/pkgs/top-level/python-packages.nix
@@ -4736,6 +4736,8 @@ in {
     };
   };
 
+  fritzconnection = callPackage ../development/python-modules/fritzconnection { };
+
   frozendict = buildPythonPackage rec {
     name = "frozendict-0.5";
 

